packages/network/src/chart/Nodes.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import { Group } from '@vx/group';
+
+const propTypes = {
+  nodes: PropTypes.array.isRequired,
+  nodeComponent: PropTypes.func.isRequired,
+  className: PropTypes.string,
+};
+
+const defaultProps = {
+  className: '',
+};
+
+export default function Nodes({
+  nodes,
+  nodeComponent,
+  className,
+  ...rest
```

Comment:
I think this is nice in `vx` where the user can do whatever they want, but since you're not exposing everything in `Network` I would just define the mouse event handlers, etc. specifically here like you do in `Node.jsx`